sunpy/map/sources/sdo.py

```diff
@@ -188,14 +207,15 @@ def scale(self):
     def date(self):
         """
         Image observation time.
+        """
+        return self._get_date('IMG_FRST')
```

Comment:
I was not sure what to use for this.

Reply:
Assuming we don't get actual input from an HMI person, we should stare at bunch of the HMI files to see whether this is an appropriate choice

Reply:
I should just send an email.